pyproject.toml

```diff
@@ -89,3 +89,7 @@ ignored-argument-names = "_.*|^ignored_|^unused_|args|kwargs"
 [tool.bandit]
 exclude_dirs = ["tests"]
 skips = ["B101"]
+
+authors = [
+    "J. David Ibáñez <jdavid.ibp@gmail.com>",
+]
```

Comment:
I suppose you didn't mean to add it to the `bandit` section, but to the `project` section. We use dependencies that David maintains but I don't think David ever contributed to dvc-data, so I don't quite understand why you are adding him to the authors.